core/storage/feedback/gae_models_test.py

```diff
@@ -32,6 +32,9 @@ def test_put_function(self):
         feedback_thread_model = feedback_models.FeedbackThreadModel(
             exploration_id='exp_id_1')
         feedback_thread_model.put()
+        feedback_thread_model = feedback_models.GeneralFeedbackThreadModel(
+            entity_type='exploration', entity_id='exp_id_1')
```

Comment:
'exploration' should probably be a named constant

Reply:
Done